### ISSUE TITLE
Latest version of JetPack seems to cause issues with Shopp

### ISSUE THREAD

**Issue by sierrab123** [ingenesis/shopp#3376]
Just saw a site that lost all of the products on the storefront page and various category pages (not all).

Reverting to an earlier version of Jetpack resolved the missing products.

Anyone else seeing this?

**Comment by adamsewell** [comment]
Yes, I'm seeing the same thing. It's got something to do with the remove_filter in ShoppCollectionPage::content. 

If you comment out ```remove_filter('the_content', array($this, 'content'), 20);``` in that function it seems to work. I can't figure out what the purpose of this line is in the first place though. Maybe @jonathandavis or @clifgriffin can comment on what this was intended for.

**Comment by Just-m** [comment]
Did you try changing 20 to 99 or something?

**Comment by adamsewell** [comment]
No, but it would theoretically do the same. Unless I'm missing something, I really don't think it should be there at all.

**Comment by sewmyheadon** [comment]
We had the exact same issue happen on a client's site and I was able to troubleshoot it to two specific modules in the JetPack plugin: *Sharing* and *Publicize*.  If either of these modules are active, the catalog page content disappears, but deactivating those modules, while leaving JetPack active solved the problem.

Since we need those add-ons, our longer-term fix was to roll JetPack back to the last known compatible version (3.6.1).
